tools/ci/tc/sink_task.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import logging
+import os
+
+import taskcluster
+
+
+logging.basicConfig()
+logger = logging.getLogger()
+
+
+def check_task_statuses(task_ids):
+    """Verifies whether a set of Taskcluster tasks completed successfully or not.
+
+    Returns 0 if all tasks passed completed successfully, 1 otherwise."""
+
+    queue = taskcluster.Queue({'rootUrl': os.environ['TASKCLUSTER_PROXY_URL']})
```

Comment:
Could you add a comment here explaining why we need to use the proxy URL instead of the root URL?

Reply:
So I actually don't know the concrete reason why (following decision.py); I assume its because we don't have access to or don't want to put credentials here? (Based on https://docs.taskcluster.net/docs/manual/design/env-vars#taskcluster_proxy_url and https://github.com/taskcluster/taskcluster/tree/master/tools/taskcluster-proxy)

Reply:
Thanks. I thought these queries were not ACL'ed.

Reply:
I think they aren't and just using the TC URL should be fine.

Reply:
Based on https://github.com/web-platform-tests/wpt-tc-checks/pull/9/files, looks like we can do so indeed. Fixed.